### ISSUE TITLE
Nail down table of contents

### ISSUE THREAD

**Issue by franciscothebrady** [franciscothebrady/R4SS#1]
1.  intro: what is the aim of the book 
    -  Why do social science practitioners need programming and statistical skills
    - etc
2.  introduction to R: beginning steps
    - begin with case study of social science data project 
    - introduction to data types in R 
    - extracting data stored in different formats (pdfs, csv, etc)
3.  statistical concepts for effective for social science research
    - descriptive statistics: what we know
    - correlation - connecting the dots
    - causal inference - establishing a direction
    - external validity - controlling for confounders (maybe move to later chapter) 
4.  longer projects:
    - version control: git
    - reproducable research 
    -  scandal!
5.

**Comment by franciscothebrady** [comment]
very preliminary

**Comment by damianrt** [comment]
Possible Topics or sections

Carry out various types of data analysis: qualitative, quantitative

Data Structures as they relate to social science concepts

* structured: tabular, hierarchical
* Unstructured, text, images, other
* univariate (vectors)
* Multivariate (data frames, Matrices)

Statistical tests: t, Chisq, Anova

Modelling: Regression, lm, Glm, Fitting arbitrary models, Optim

The Formal empirical research process:

1. Ask a question
2. Formulate a hypothesis. Make a prediction based on assumed relationships between measurable concepts, aka theory.
3. Collect data
4. Analyze data
5. Evaluate Hypothesis and refine theory if necessary

**Comment by franciscothebrady** [comment]
Some people to shamelessly steal from:  
+  [Urban Informatics Toolkit](https://ui.josiahparry.com/)  
+  [Computing for the Social Sciences](https://cfss.uchicago.edu/) - In particular the section on shapefiles.

**Comment by franciscothebrady** [comment]
Data Acquisition (Working with APIs):
https://data.library.virginia.edu/using-census-data-api-with-r/

**Comment by franciscothebrady** [comment]
@damianrt just wrote the git addendum

**Comment by franciscothebrady** [comment]
https://mgimond.github.io/Spatial/reading-and-writing-spatial-data-in-r.html

**Comment by damianrt** [comment]
In addition to the data structures R uses to store data, we should also cover the various types of variables from a data analysis perspective: numerical vs categorical, continuous vs discrete, ordered and unordered.

**Comment by franciscothebrady** [comment]
I think variable types comes before data structures right ?

**Comment by damianrt** [comment]
I agree. I think it is useful to introduce vectors and the data types before introducing data frames etc. if we want to be able to introduce programming or more advanced usage later.
